backend/tests/test_func_login_with_db.py

```diff
@@ -0,0 +1,12 @@
+from tools.create_db_for_tests import create_test_app_with_db
+from tools.work_with_db import get_psw_from_db
+
+
+def test_get_log_psw_from_db():
+    lst_log_psw = ['admin', 'Pyth0n']
```

Comment:
it would be better to create two separate variables here instead. 
The list as a whole is unused, you always unpack it. Therefore it's better to name the entities of that list.

Reply:
our header decode function returns such a list. I thought it would be clearer